CoEpi/api/Api.swift

```diff
@@ -50,18 +50,16 @@ class ApiImpl: Api {
     }
 
     func getCenKeys() -> Single<[String]> {
-        let unixDT = Int(currentAttemptedExposureCheck.timeIntervalSince1970)
-        let stringURL = "https://coepi.wolk.com:8080/cenkeys/\(unixDT)"
+        let stringURL = "https://coepi.wolk.com:8080/cenkeys/\(Int(currentAttemptedExposureCheck.timeIntervalSince1970))"
```

Comment:
Let's remove the timestamp for now. It's not required.

And remove the variables from `Api`.